src/azure-cli/azure/cli/command_modules/managedservices/__init__.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core import AzCommandsLoader
+from azure.cli.core.profiles import ResourceType
+from azure.cli.command_modules.managedservices._help import helps  # pylint: disable=unused-import
+from azure.cli.core.commands import AzCommandGroup
+
+
+class ManagedServicesCommandGroup(AzCommandGroup):
+
+    def managedservices_custom(self, name, method_name=None, **kwargs):
+        command = self.command_loader.command_table[self.custom_command(name, method_name, **kwargs)]
+        if not command:
+            return
```

Comment:
This custom command group does nothing and should be removed.

Reply:
removed